service/controller/resource/app/delete.go

```diff
@@ -131,7 +131,7 @@ func (r Resource) waitForAppDeletion(ctx context.Context, cr capi.Cluster, appNa
 		r.logger.Errorf(ctx, err, "retrying in %s", t)
 	}
 
-	b := backoff.NewExponential(15*time.Second, 5*time.Second)
+	b := backoff.NewConstant(15*time.Second, 5*time.Second)
```

Comment:
We use a short 15 sec backoff while the app-operator and chart-operator app CRs are deleted. 

So using a constant backoff makes more sense.